packages/client/CHANGELOG.md

```diff
@@ -562,5 +562,3 @@ showing the relations between the main components as well as the initialization
 ## [0.0.1] - 2018-10-26
 
 - Initial development release
-
-[0.0.1]: https://github.com/ethereumjs/ethereumjs-client/tree/v0.0.1
```

Comment:
This URL is removed? (but URL still works?)

Reply:
Removed by mistake, have reverted.